src/main/java/uk/nhs/hee/tis/revalidation/connection/service/ExceptionLogService.java

```diff
@@ -38,7 +38,7 @@
 @Slf4j
 @Transactional
 @Service
-public class ExceptionService {
+public class ExceptionLogService {
 
   @Autowired
   private ExceptionRepository repository;
```

Comment:
If you refactor this to be constructor injection instead of autowired, it will solve the code smells push the test coverage up

Reply:
Constructor injection is used. Thanks